src/sentry/static/sentry/app/components/events/relatedEvents/emptyState.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import {t} from 'app/locale';
+import EmptyStateWarning from 'app/components/emptyStateWarning';
+import {Panel} from 'app/components/panels';
+
+type Props = {
+  message?: string;
+};
+
+const EmptyState = ({message}: Props) => (
+  <Panel>
+    <EmptyStateWarning small>
+      {message ??
+        t(
+          'No related events have been found 12 hours either before and after the occurrence of this event.'
```

Comment:
Would showing the time range that we checked be helpful here?

Reply:
I've updated it...Now I am displaying something like:

![image](https://user-images.githubusercontent.com/29228205/90613705-09315600-e20a-11ea-852d-ccdd38a0e39b.png)